### PR TITLE
Issue #3057538: Update to blazy 2.x and slick 2.x

### PR DESCRIPTION
**Pull request by chrfritsch**
Make sure these boxes are checked before submitting your pull request - thank you!

- [ ] All coding styles are fulfilled. ([How to check for cs issues?](https://github.com/BurdaMagazinOrg/thunder-dev-tools/blob/master/README.md#code-style-guidelines))
- [ ] All tests are running locally. ([How to run the test?](https://github.com/thunder/thunder-distribution/blob/8.x-3.x/docs/development.md#how-to-run-the-tests))
- [ ] Necessary update hooks are provided.
- [ ] User roles have correct access for new introduced permission.
- [ ] Every thunder module has a README.md in its root. Follow [this guidelines](https://www.drupal.org/node/2181737), but we don't need every topic.
- [ ] Code is covered with well-balanced amount of inline comments.

If you are really awesome, then your feature is covered by additional tests. Well done!

### CODE REVIEW

**Comment by chrfritsch** [comment]
This patch is needed https://www.drupal.org/project/drupal/issues/2863986

File: composer.json [hunk 1]
```diff
@@ -80,6 +80,9 @@
             "drupal/linkit": {
                 "Drupal 8.7 no longer has a dedicated canonical route for media entities, breaks entity matcher deriver": "https://www.drupal.org/files/issues/2019-05-03/linkit-media-87-3040749-8.patch"
             },
+            "drupal/core": {
+                "Allow updating modules with new service dependencies": "https://www.drupal.org/files/issues/2018-11-15/2863986-77-8.7.x.patch"
+            },
```
Comment (dbosen) [inline composer.json:85]: Add strict core version require.

File: drupal-org.make [hunk 2]
```diff
@@ -113,7 +113,7 @@ projects[simple_sitemap][version] = 3.2
 projects[simple_sitemap][patch][] = https://www.drupal.org/files/issues/2019-06-25/3063870-3.patch
 projects[simple_sitemap][patch][] = https://www.drupal.org/files/issues/2019-06-24/3062747-8.patch
 projects[slick][type] = module
-projects[slick][version] = 1
+projects[slick][version] = 2
```
Comment (dbosen) [inline drupal-org.make:116]: core patch is missing in make file

**Review by dbosen — CHANGES_REQUESTED** [review]

**Comment by chrfritsch** [comment]
Hopefully, we can remove the patch with Drupal 8.7.5

**Review by dbosen — APPROVED** [review]

**Pull request merged**